### DISCUSSION TITLE
After reboot my local rule missing

### DISCUSSION THREAD

**Discussion by MarioBabic** [Security-Onion-Solutions/securityonion#8892] [Unsupported Versions]
I properly configure local rules. After running the commands sudo so-rule-update and sudo salt \* state.highstate local rules are copied to all.rules and work without problems. However, after that, my rules are deleted from local.rules and after restarting the server, those rules are not in all.rules.

**Comment by dougburks** [comment]
What is the output of the following?
```
sudo cat /opt/so/saltstack/local/salt/idstools/local.rules
```

**Comment by MarioBabic** [comment]
Output is:

 #Custom Suricata rules go in this file

**Reply by dougburks** [reply]
Did you add your rules to `/opt/so/saltstack/local/salt/idstools/local.rules` as shown at https://docs.securityonion.net/en/2.3/local-rules.html?
